drivers/hmis/app/graphql/types/hmis_schema/ce_referral_source_enrollment.rb

```diff
@@ -0,0 +1,132 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeReferralSourceEnrollment < Types::BaseObject
+    # object is an OpenStruct with the following shape:
+    # {
+    #   enrollment: Hmis::Hud::Enrollment,
```

Comment:
Can we add a big call-out here stating that the Enrollment may be from another data source (HMIS or otherwise). Maybe I'm getting ahead of myself to another ticket, if you're doing that on another branch its all good